sherlock_homes.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |spec|
 
   # Runtime
   spec.add_dependency 'geocoder', '~> 1.2'
+  spec.add_dependency 'activesupport', '~> 4.2'
```

Comment:
at this moment we using only the `core_ext` I mean `"string".present?` `"mulit string".underscore`